### ISSUE TITLE
unable to build aar's with local engine

### ISSUE THREAD

**Issue by gaaclarke** [flutter/flutter#93512]
error:
```
$ flutter --local-engine-src-path ~/dev/engine/src --local-engine=android_debug_unopt_arm64 build aar

💪 Building with sound null safety 💪

Running Gradle task 'assembleAarDebug'...                           3.4s


FAILURE: Build failed with an exception.

* Where:
Initialization script '/Users/aaclarke/dev/flutter/packages/flutter_tools/gradle/aar_init_script.gradle' line: 167

* What went wrong:
java.lang.NullPointerException (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s


Gradle task assembleAarDebug failed with exit code 1.
```

flutter verison:
```
[✓] Flutter (Channel unknown, 2.6.0-12.0.pre.668, on macOS 11.6.1 20G224 darwin-x64, locale en-US)
    • Flutter version 2.6.0-12.0.pre.668 at /Users/aaclarke/dev/flutter
    • Upstream repository unknown
    • Framework revision 2ea8b497da (29 hours ago), 2021-11-10 10:10:14 -0800
    • Engine revision 766f3a5bec
    • Dart version 2.15.0 (build 2.15.0-285.0.dev)
```

cc @blasten

**Comment by eggfly** [comment]
Same issue before but now I'm ok now using the newest flutter and local engine to build aar. @gaaclarke

**Comment by wangying3426** [comment]
I also encountered the same problem, may I ask which commit has fixed this problem?  My engine version is 2.10.2
